.github/workflows/ios-maestro.yml

```
@@ -100,46 +115,225 @@ jobs:
             -   name: Install dependencies
                 run: yarn install --immutable
 
-            -   name: Verify Xcode toolchain
-                run: xcodebuild -version
+            -   name: Select and assert Xcode toolchain
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)